pandas/core/groupby/ops.py

```diff
@@ -494,6 +519,17 @@ def _cython_operation(
                 values = ensure_int_or_float(values)
         elif is_numeric and not is_complex_dtype(values):
             values = ensure_float64(values)
+        elif is_categorical:
+            if how in self._cat_method_blacklist:
```

Comment:
I really don't like doing this. Can you elaborate when we can actually process this? listing methods is a bad idea generally.

Reply:
I'm not exactly sure what you mean by "when we can actually process this" this but I agree that listing methods isn't necessarily thorough and isn't robust. However, I've been unable to find a suitable alternative to blacklisting methods where we don't want to apply the aggregation on the category codes -- open to other ideas though. Please find more context in [this](https://github.com/pandas-dev/pandas/pull/33739#discussion_r415151884) thread.